exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/statements/InsertStatement.kt

```diff
@@ -132,6 +132,9 @@ open class InsertStatement<Key:Any>(val table: Table, val isIgnore: Boolean = fa
             // http://viralpatel.net/blogs/oracle-java-jdbc-get-primary-key-insert-sql/
             transaction.connection.prepareStatement(sql, autoIncColumns.map { it.name.inProperCase() }.toTypedArray())
 
+        nextValColumns.isNotEmpty() ->
```

Comment:
Is it possible to mix autoincrement and sequence columns in a single query and get both in a result?

Reply:
Yes, there are two possible ways to mix them : 
- put autoincrement and sequence columns in autoIncColumns variable.
- or change it like this : prepareStatement(sql, **(autoIncColumns + nextValColumns)**.map { it.name.inProperCase() }.toTypedArray()).
Which one do you think is better?